openmc/mgxs_library.py

```diff
@@ -67,22 +71,7 @@ class XSdata(object):
         is "angle".
     num_polar : int
         Number of equal width angular bins that the polar angular domain is
-        subdivided into. This only applies when :attr:`XSdata.representation`
-        is "angle".
-    use_chi : bool
-        Whether or not a chi vector or nu-fission matrix was used.
-    vector_shape : iterable of int
```

Comment:
Thanks, it looks pretty cool this way and the string key is self explanatory too, which I like